src/test/java/se/nackademin/LemurTest.java

```diff
@@ -0,0 +1,72 @@
+package se.nackademin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class LemurTest {
+    @Test
+    public void shouldEat() {
+        Lemur tomislav = new Lemur();
+        tomislav.hunger = 8;
+        boolean isHungry = tomislav.eat();
+        assertTrue(isHungry);
+
+    }
+    @Test
+    public void shouldNotEat() {
+        Lemur tomislav = new Lemur();
+        tomislav.hunger = 3;
+        boolean isHungry = tomislav.eat();
+        assertFalse(isHungry);
+
+    }
+    @Test
+    public void shouldSleep() {
+        Lemur alexander = new Lemur();
+        alexander.energyLevel = 2;
+        boolean isTired = alexander.sleep();
+        assertTrue(isTired);
+    }
+    @Test
+    public void shouldSnuggle() {
+        Lemur timmy = new Lemur();
+        String response = timmy.snuggle();
+        assertEquals("Something went wrong.", "Snuggling...", response);
+        assertEquals(8, timmy.energyLevel);
+        assertEquals(6, timmy.getFluffiness());
+    }
+    @Test
+    public void shouldSpeak() {
+        Lemur mira = new Lemur();
+        String response = mira.speak();
+        assertEquals("Hakkunah Matatatatatatata", response);
+    }
+    @Test
+    public void shouldBackflip() {
+        Lemur lemur = new Lemur();
+        String response = lemur.backflip();
+        assertEquals("Lemur backflips", response);
+    }
+    @Test
+    public void shouldGetOld() {
```

Comment:
This and below tests aren't really testing that getters works, they are more testing that certain defaul values are set, such as age == 1. It's fine to test that default values are set to something specific, but I would name the test more like "defaulAgeShouldBe1" or similar.